package.json

```diff
@@ -115,10 +117,8 @@
     "*.{js,jsx,ts,tsx,mjs}": [
       "eslint --fix"
     ],
-    "*.css": [
-      "stylelint --allow-empty-input --fix"
-    ],
     "*": [
+      "stylelint --allow-empty-input --fix",
```

Comment:
I'm still skeptical about using stylelint to fix non-CSS code though, don't like any of the diff in the MD files.

Reply:
what do you dislike?

I don't have a strong opinion on this but we already use similar rgb strings in our theme CSS files

```css
.copyButton {
  background: rgb(0 0 0 / 30%);
}
```

Reply:
Fair enough, I just feel like what we present to the user doesn't necessarily have to follow our internal code style, but rather what's most popular (and I guess the legacy function style is still more widely recognized)... I'm fine with both.

Reply:
I agree on that, and also found this syntax less popular

maybe we should just change the rules, so that the most popular syntax also applied to our theme (hence swizzling too)

Reply:
But I do like the modern syntax better :P Don't really want to tweak our code style, just want to enforce control over our text content. ESLint doesn't enforce anything on Markdown code blocks.

Reply:
I'm fine if we disable this on all md files or just this specific rule, that doesn't seem to have a huge impact anyway

Reply:
Yeah, I don't really want to use it to lint anything other than CSS. We have Prettier to format the inline code anyways.